static/src/javascripts/bootstraps/enhanced/common.js

```diff
@@ -120,6 +121,18 @@ const loadAnalytics = (): void => {
     }
 };
 
+const loadGoogleAnalytics = (): void => {
+    const handleGoogleAnalytics = (gaConsents: boolean): void => {
+        if (gaConsents && !window.ga.loaded) {
+            window.initialiseGa()
+        } else {
+            // prevent console errors from code calling window.ga for sessions where we have not loaded the tracking script
+            window.ga = function() {}
```

Comment:
Because you're defining this here https://github.com/guardian/frontend/pull/23203/files#diff-55b65cfe67f24b5a3598c9d4bbd50fb8b8e4ad5dd5d2bdbf557ee66a61875201R18 do you need the `else` here? is this to *reset* `window.ga` if consent changes to false (Maybe a small comment to explain that if the case)?

Reply:
It is, yes - let me change the comment